pytest_redis.py

```diff
@@ -54,12 +56,59 @@ def pytest_collection(session, genitems=True):
     return items
 
 
+def get_redis_connection(config):
+    """Get a redis connection base on config args."""
+    redis_host = config.getoption('redis_host')
+    redis_port = config.getoption('redis_port')
+    r_client = redis.StrictRedis(host=redis_host,
+                                 port=redis_port)
+    return r_client
+
+
+def determine_redis_list_key_to_use(session, redis_connection):
```

Comment:
Maybe rename this... You always consume from the main list, you just sometimes have to populate it first 😉